src/components/EventCreationPage/SelectDateModal.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react";
+import Modal from "@material-ui/core/Modal";
+import Button from "@material-ui/core/Button";
+import Container from "@material-ui/core/Container";
+import Grid from "@material-ui/core/Grid";
+import { makeStyles } from "@material-ui/core/styles";
+import { createMuiTheme } from "@material-ui/core";
+import { ThemeProvider } from "@material-ui/styles";
+import { MuiPickersUtilsProvider, DatePicker } from "@material-ui/pickers";
+import DateFnsUtils from "@date-io/date-fns";
+import Typography from "@material-ui/core/Typography";
+
+const SelectDateModal = ({
+  open,
+  handleClose,
+  eventDate,
+  setEventDate,
+}: {
+  open: boolean;
+  handleClose: () => any;
+  eventDate: Date | null;
+  setEventDate: (date: any) => void;
+}) => {
+  const classes = useModalStyles();
+  const [date, setDate] = useState<any>(eventDate);
+  return (
+    <Modal open={open} onClose={handleClose}>
+      <Container classes={{ root: classes.root }}>
+        <Typography classes={{ root: classes.text }}>
+          Select Event Date:
+        </Typography>
```

Comment:
This text alignment doesn't match the design (basing it off screenshot on the task card).